configuration.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: Configuration Files
+description: Configure your Chalk project.
+---
+
+import {
+    Attribute,
+    AttributeTable,
+    SubAttribute,
+    SubAttributeTable,
+} from '@/components/AttributeTable'
+
+---
+
+## Overview
+
+Your Chalk project's configuration is shared across the following files:
+
+1. `chalk.yaml` (or `chalk.yml`): Configuration for your project's deployment
+1. `.chalkignore`: Files to exclude from your project's deployment
+1. `requirements.txt`: Default Python requirements for Chalk to install via `pip` (can be overridden in `chalk.yaml`)
+
+Here's our recommended repository structure:
+
+```
+company_chalk/
+├── src/
+│  ├── resolvers/
+│  │  ├── .../
+│  │  ├── __init__.py
+│  │  └── pipelines.py
+│  ├── __init__.py
+│  ├── datasources.py
+│  └── feature_sets.py
+├── tests/
+│  └── ...
+├── notebooks/
+│  └── ...
+├── .chalkignore
+├── chalk.yaml
+├── README.md
+└── requirements.txt
+```
+
+## chalk.yaml
+
+Use `chalk.yaml` to configure your project's Docker environment, Python configuration, and feature metadata validation.
+
+### Schema
+<AttributeTable title={"chalk.yaml"}>
+<Attribute field='project' kind='str'>
+  The name of this Chalk project, which must match your project's name in the dashboard (case-sensitive).
+</Attribute>
+<Attribute field='environments' kind='dictionary'>
+  Per-environment overrides. See our <a href="/docs/docker">Docker documentation</a> for more details.
+  <SubAttributeTable title=''>
+    <SubAttribute field='[environment_name]' kind='dictionary'>
+      The name of this environment, such as <code>prod</code> or <code>qa</code>. Find available environment names in
+      your dashboard. Use <code>default</code> to configure default values to apply to all of this project's
+      environments.
+      <SubAttributeTable title=''>
+        <SubAttribute field='runtime' kind='"python310" | "python311"'>
+          The Python version for the Chalk project.
+        </SubAttribute>
+        <SubAttribute field='requirements' kind='str'>
+          Python requirements file to install with <code>pip</code>.
+        </SubAttribute>
+        <SubAttribute field='dockerfile' kind='str'>
+          Path to an alternative Dockerfile that will be used to build a base image for your Chalk deploys.
+        </SubAttribute>
+      </SubAttributeTable>
+    </SubAttribute>
+  </SubAttributeTable>
+</Attribute>
+<Attribute field='validation' kind='dictionary'>
+  Configuration for metadata validation for features. See our <a href="/docs/validation#feature-validation-requirements">validation
+  documentation</a> for more details.
+  <SubAttributeTable title=''>
+    <SubAttribute field='[feature_name]' kind='dictionary'>
```

Comment:
i think this is just a section for all defaults and all resolvers: 

so instead of [feature name] it's feature

and instead of [resolver name] it'd be resolver

Reply:
if you look at this example: 

```
validation:
  feature:
    metadata:
      - name: owner
        missing: error
      - name: description
        missing: error
      - name: tags
        missing: error
  resolver:
    metadata:
      - name: owner
        missing: error
      - name: description
        missing: error
```

Reply:
![image](https://github.com/user-attachments/assets/df3c3b55-bbc0-4442-9183-89a736d2412c)